kedro/framework/cli/starters.py

```diff
@@ -553,3 +587,19 @@ def _validate_config_file(config: dict[str, str], prompts: dict[str, Any]):
             f"'{config['output_dir']}' is not a valid output directory. "
             "It must be a relative or absolute path to an existing directory."
         )
+
+
+def _validate_config_file_inputs(config: dict[str, str]):
```

Comment:
I'm not quite sure this function is actually adding much. The messages don't seem to appear to the user at all and the regex checking is the same as specified in `prompts.yml`. What was the reason you added it?